.gitignore

```diff
@@ -0,0 +1 @@
+/build
```

Comment:
see https://github.com/visenze/productcat-sdk-android/blob/master/.gitignore and update the list.

Reply:
ok

Reply:
have u pushed the changes? this is not updated

Reply:
pushed now